packages/react/src/Testimonial/Testimonial.stories.tsx

```diff
@@ -9,6 +9,7 @@ export default {
   component: Testimonial,
   args: {
     align: 'start',
+    quoteMarkFill: 'var(--brand-color-text-primary)',
```

Comment:
FYI 👇 

![Screenshot 2023-08-15 at 10 44 47](https://github.com/primer/brand/assets/13340707/f6890eaa-48e1-4f56-9ff2-23fa56de74a4)

Could we set the default value in the design token layer also?

```suggestion
    quoteMarkFill: 'var(--brand-color-text-default)',
```